runlogwatch.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/bash
+
+# Ramdisk logs path
+LOG_DIR="/shared/log/ironic/deploy"
+
+inotifywait -m "${LOG_DIR}" -e create -e moved_to |
```

Comment:
Do we want to watch for `create` or `close_write` events?

Reply:
Agreed, will replace `create` with `close_write` in the next version.